src/plugins/data/public/ui/filter_bar/filter_editor/value_input_type.test.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
+import { ValueInputType } from './value_input_type';
+
+let onChangeMock: any;
+
+describe('Value input type', () => {
+  beforeAll(() => {
+    onChangeMock = jest.fn();
+  });
+  it('is number', async () => {
+    const valueInputProps = {
+      value: 1,
```

Comment:
For BigInt, use exactly the same test and only change value to `1n`.

Reply:
added a unit test for bigint using 1n